src/components/InterestForm/index.tsx

```diff
@@ -1,73 +1,101 @@
 'use client';
 
-import { useState, useEffect } from 'react';
 import { upsertInterest } from '@/api/supabase/queries/interest';
-import { Interest, CaseListing, RoleEnum } from '@/types/schema';
+import Button from '@/components/Button';
+import DateInput from '@/components/DateInput';
+import RadioGroup from '@/components/RadioGroup';
+import TextAreaInput from '@/components/TextAreaInput';
+import COLORS from '@/styles/colors';
+import { Flex } from '@/styles/containers';
+import { H3, P } from '@/styles/text';
+import { Interest, Listing } from '@/types/schema';
 import { useAuth } from '@/utils/AuthProvider';
 import { isValidDate } from '@/utils/helpers';
-import { P, H3 } from '@/styles/text';
-import COLORS from '@/styles/colors';
-import DateInput from '../DateInput';
-import Button from '../Button';
-import {
-  FormContainer,
-  FormTextArea,
-  FormQuestion,
-  RadioGroup,
-  Radio,
-  RadioInput,
-  FormFooter,
-  FormWarning,
-  ErrorText,
-  RadioLabel,
-} from './styles';
+import { useEffect, useState } from 'react';
+import { EmptySpace, FormContainer, FormFooter, FormWarning } from './styles';
```

Comment:
instead of having these as individual imports, it may be good to do `import * as Styles from './styles';` just to the markup clearer as to what components are local styles

Reply:
sg, done